components/bsx/Create/CreateCollection.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div>
+    <Loader v-model="isLoading" :status="status" />
+    <BaseCollectionForm v-bind.sync="base">
+      <template v-slot:main>
+        <b-field class="mb-5" />
+      </template>
+      <template v-slot:footer>
+        <CustomAttributeInput
+          :max="10"
+          v-model="attributes"
+          class="mb-3"
+          visible="collapse.collection.attributes.show"
+          hidden="collapse.collection.attributes.hide" />
```

Comment:
What do you mean?

Reply:
why do we get `visible` and `hide` at the same time? If `show` is `false` it would be hidden already.

[avoiding collapse](https://stackoverflow.com/questions/133051/what-is-the-difference-between-visibilityhidden-and-displaynone)

Reply:
If you would have 10+ attributes it would be supper messy :) 
<img width="495" alt="Screenshot 2022-05-26 at 16 29 07" src="https://user-images.githubusercontent.com/22471030/170508927-fc8f3bdc-b7d3-4274-9d75-9389f08ce7b9.png">

Reply:
Oh, okii 😄